### ISSUE TITLE
Accessibility: website, date range and segment selectors should be usable with the keyboard

### ISSUE THREAD

**Issue by mnapoli** [matomo-org/matomo#7006]
refs #4167

The website selector, date range selector and segment selector are not usable with the keyboard. Screen readers read them as raw text (instead of a form control) which makes it impossible to use them without the mouse.

**Comment by tassoman** [comment]
I've found this useful link: http://accessibility.psu.edu/tableshtml

**Comment by tassoman** [comment]
I've tested using NVDA, i can move inside the Date Range DIV, but unable to select date range radio buttons. Only "apply date range" could be selected.

**Comment by jdeyla** [comment]
I can't use any control on div.top_controls with the keyboard on demo.piwik, maybe I'm testing the wrong version?

**Comment by mnapoli** [comment]
The demo is the latest version, so you are testing on the correct one. There is definitely room for improvement here.

**Comment by mattab** [comment]
@jdeyla basically this issue is opened which means "there is a problem" - if anyone can join the accessibility work and help us improve accessibility of features like Date Range, Segment selector, this would be amazing!

**Comment by tassoman** [comment]
Probably we could work on it in the next days. I'll send a pull request :package:

**Comment by CheweyZ** [comment]
This would appear to be something that i added a last pr and has been remade in #11252 so it should be resolved?

**Comment by mattab** [comment]
Great work @CheweyZ - Thanks for your help getting this resolved!
